utils/create_srg_export.py

```diff
@@ -299,7 +301,33 @@ def html_plain_text(source: str) -> str:
     return result
 
 
-def replace_variables(source: str, variables: dict) -> str:
+def get_variable_value(root_path: str, product: str, name: str, selector: str) -> str:
+    product_value_full_path = pathlib.Path(root_path).joinpath('build').joinpath(product)\
+        .joinpath('values').joinpath(f'{name}.yml').absolute()
+    if not product_value_full_path.exists():
+        sys.stderr.write(f'Undefined variable {name}\n')
+        exit(7)
+    with open(product_value_full_path, 'r') as f:
+        var_yaml = yaml.load(Loader=yaml.SafeLoader, stream=f)
+        if selector and selector.isnumeric():
+            selector = int(selector)
```

Comment:
We have a problem here that we have variables with left zeroes and they must be treated in a different way, otherwise 

https://github.com/ComplianceAsCode/content/blob/adb3d514317f80c0f4f2e9a31959d1362a963b86/linux_os/guide/system/accounts/accounts-session/user_umask/var_accounts_user_umask.var#L14

would be transformed into `7` and it would find the correspondent value.

Not every numeric value should be transformed into integer.

Reply:
So the issue is that we have some integer keys, see:

https://github.com/ComplianceAsCode/content/blob/adb3d514317f80c0f4f2e9a31959d1362a963b86/linux_os/guide/system/accounts/accounts-session/var_accounts_max_concurrent_login_sessions.var#L14

So it seems I have some work to get this working.

Reply:
Yes, I noticed as well this case. But for example, if you use the Loader=yaml.BaseLoader, it will treat this as `'1'` for example, as everything will be a string.

Reply:
I have updated the PR with `BaseLoader`.